acceptance/acceptance_test.go

```diff
@@ -65,6 +66,10 @@ var Scripts = map[string]bool{
 	PrepareScript:    true,
 }
 
+var Ignored = map[string]bool{
```

Comment:
Does it make sense to make this configurable, maybe in the future?

Reply:
We can discuss it if we have use case for it.